public/pages/EditFeatures/components/AggregationSelector/__tests__/AggregationSelectior.test.tsx

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
typo: rename file to AggregationSelector.test.tsx

Reply:
sure, will change